src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
+import machines.Bulldozer;
+import machines.Excavator;
+import machines.Machine;
+import machines.Truck;
+
 public class MainApp {
 
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
```

Comment:
we should use abstraction instead of implementation
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#use-abstract-reference-instead-of-specific-one-where-possible:~:text=Use%20abstract%20reference%20instead%20of%20specific%20one%20where%20possible